DuckDuckGo/OnboardingViewController.swift

```diff
@@ -21,10 +21,14 @@ import UIKit
 import Core
 
 class OnboardingViewController: UIViewController, Onboarding {
-    
-    private var controllerNames: [String] = {
-        if #available(iOS 14.0, *) {
+        
+    private lazy var controllerNames: [String] = {
+        if variantManager.isSupported(feature: .onboardingHomeRow) {
+            return ["onboardingHomeRow"]
```

Comment:
Technically redundant, right?  But don't feel strongly either way, because this is also explicit which is good.

Reply:
Yeah, totally redundant, but I preferred the explicitness of it.